plutus-pab/src/Cardano/Wallet/Mock/Types.hs

```diff
@@ -51,31 +52,37 @@ import Ledger.Ada (Ada)
 import Plutus.ChainIndex (ChainIndexQueryEffect)
 import Plutus.PAB.Arbitrary ()
 import Plutus.PAB.Types (PABError)
+import Plutus.V1.Ledger.Api (Address)
 import Prettyprinter (Pretty (pretty), (<+>))
 import Servant (ServerError)
 import Servant.Client (ClientError)
 import Servant.Client.Internal.HttpClient (ClientEnv)
 import Wallet.Effects (NodeClientEffect, WalletEffect)
 import Wallet.Emulator.Error (WalletAPIError)
 import Wallet.Emulator.LogMessages (RequestHandlerLogMsg, TxBalanceMsg)
-import Wallet.Emulator.Wallet (Wallet, WalletId, WalletState (WalletState, _mockWallet), mockWalletPaymentPubKeyHash,
-                               toMockWallet)
+import Wallet.Emulator.Wallet (Wallet, WalletId, WalletState (WalletState, _mockWallet), mockWalletAddress,
+                               mockWalletPaymentPubKeyHash, toMockWallet)
 
 -- | Information about an emulated wallet.
 data WalletInfo =
     WalletInfo
         { wiWallet            :: Wallet
-        , wiPaymentPubKeyHash :: PaymentPubKeyHash -- ^ Hash of the wallet's public key, serving as wallet ID
+        , wiPaymentPubKeyHash :: PaymentPubKeyHash
+        -- ^ Hash of the wallet's public key, serving as wallet ID.
+        -- TODO Remove eventually as it is replaced by 'wiAddresses'.
```

Comment:
Can we deprecate record fields? I suppose just as a function `wiPaymentPubKeyHash :: WalletInfo -> PaymentPubKeyHash`?